code/socialDistribution/templates/posts/comments.html

```diff
@@ -35,50 +35,7 @@ <h2>Comments</h2>
     <div class="mb-1">&nbsp;</div>
 </div>
 
-<<<<<<< HEAD
-{% load modal %}
-<form method="POST" enctype="multipart/form-data" action="{% url 'socialDistribution:editPost' post.id %}">
-    {% csrf_token %}
-    {% modal id="edit_post_modal" type="edit" label="Edit Post" title="Edit Post" btn="Save" postid=post.id %}
-</form>
-<form method="POST" action="{% url 'socialDistribution:deletePost' post.id %}">
-    {% csrf_token %}
-    {% modal id="delete_post_modal" type="delete" label="Delete Post" title="Delete Post" btn="Delete"%}
-</form>
-
-<script>
-    // https://stackoverflow.com/a/40233358 - Shuhad zaman "urlParam"
-    $.urlParam = function (name) {
-        var results = new RegExp('[\?&]' + name + '=([^&#]*)').exec(window.location.href);
-        
-        if (results && results.length > 1) {
-            return results[1];
-        }
-        return 0;
-    }
-
-    $(function () {
-        var param = $.urlParam('action');
-
-        if (param == 'edit') {
-            $('#edit_post_modal').modal('show');
-        } else if (param == 'delete') {
-            $('#delete_post_modal').modal('show');
-        }
-    });
-
-    $('#newCommentTextArea').on('input propertychange', function () {
-        // enable button if comment not empty
-        if (this.value.length) {
-            $('#newCommentButton').prop('disabled', false);
-        } else {
-            $('#newCommentButton').prop('disabled', true);
-        }
-    })
-</script>
-=======
 {% load static %}
     <script src="{% static 'js/comment/comments.js' %}"></script>
->>>>>>> master
```

Comment:
Not sure why this was here? Seems an uncompleted merge was checked into git somehow

Reply:
O no. I hope no one force pushed